packages/component-dashboard/client/graphql/mutations.js

```diff
@@ -0,0 +1,18 @@
+import gql from 'graphql-tag'
+// TODO: component-dashboard needs its own model or a shared gql fragment for manuscript
+import { manuscriptFragment } from '../../../../app/components/pages/SubmissionWizard/operations'
```

Comment:
I think we should create an issue to break this dependency and work on it soon. This will come up again as a problem for other components we create and tackling this sooner will only serve to make it easier in the future.

Reply:
Addressed by #1738